runtime/src/main/resources/packaged/webapps/dashboard/WEB-INF/pages/fsmview.ftl

```diff
@@ -45,6 +45,10 @@
                         <th align="left" style="border-top: none; vertical-align: middle;">Correlation Id:</th>
                         <td id="correlationId" align="left" style="border-top: none; vertical-align: middle;"></td>
                     </tr>
```

Comment:
We should also expose the workflow name, so that clients can use API which rely on the name.